database/mongodb/mongodb_test.go

```diff
@@ -215,3 +227,41 @@ func TestTransaction(t *testing.T) {
 			}
 		})
 }
+
+type replicaSetStatus struct {
+	Members []replicaMember `bson:"members"`
+}
+
+type replicaMember struct {
+	StateStr string `bson:"stateStr"`
+}
+
+func waitForReplicaInit(client *mongo.Client) error {
+	ticker := time.NewTicker(time.Second * 5)
+	defer ticker.Stop()
+	timeout := time.NewTimer(time.Second * 30)
```

Comment:
Make this configurable via an environment variable, similar to `MIGRATE_TEST_CONTAINER_BOOT_TIMEOUT` in `ParallelTest()`